RevenueCatUI/CustomerCenter/ViewModels/CustomerCenterViewModel.swift

```diff
@@ -117,8 +122,12 @@ import RevenueCat
 
     #endif
 
-    func loadScreen() async {
+    func loadScreen(shouldSync: Bool = false) async {
         do {
+            if shouldSync {
+                try await self.purchasesProvider.syncPurchases()
```

Comment:
`syncPurchases` returns a `CustomerInfo` and `loadPurchaseInformation` as well. Maybe you could add a `CustomerInfo` parameter to `loadPurchaseInformation` to prevent loading `CustomerInfo` twice unnecessarily.

Reply:
Just refactored it, so we do only one call